app/views/partials/homepage.html

```diff
@@ -83,4 +85,33 @@ <h2 class="text-center">Les aides simulées</h2>
       </div>
     </div>
   </div>
+  <div class="frame-aides-homepage">
+    <h2 class="text-center">Prestations locales</h2>
+    <div ng-repeat="partenaireLocal in partenairesLocaux">
+      <hr>
+      <div class="row">
+        <div class="text-center col-md-2">
+          <img ng-src="/img/{{ partenaireLocal.imgSrc }}" alt=""/>
+          <h2>{{ partenaireLocal.label }}</h2>
+        </div>
+        <div class="col-md-10">
+          <div ng-repeat="droit in partenaireLocal.prestations">
+            <div class="clearfix" ng-if="showDetails && $index % 2 == 0"></div>
```

Comment:
Good. Then I’m fine with it.

> What is your process to check about responsiveness ?

I much prefer Safari’s [Responsive Design Mode](https://developer.apple.com/videos/play/wwdc2015/505/ <https://developer.apple.com/videos/play/wwdc2015/505/>), but both are fine to make a quick check when introducing design changes. Then, every once in a while (or for major changes) you make a smoke test with whichever devices are available around you. Just put them on the same local network as your development machine and access it through its IP  :)